APR.toc

```diff
@@ -1,8 +1,9 @@
-## Interface: 100007
+## Interface: 100100
+## X-Min-Interface: 100007
 
 ## Title: Azeroth Pilot Reloaded
 ## Notes: Continuation, optimization, and rewrite of Azeroth Auto Pilot.
-## Author: Rycia, Deathmessinger, zyrrael, Neogeekmo
+## Author: Neogeekmo, Rycia, Pahonix, Deathmessinger (legacy: zyrrael)
```

Comment:
Same as from APR-Core, Death has not contributed since at least December, they can be moved to the Legacy area, or removed entirely?

Reply:
same